.github/workflows/beam_PostCommit_Java_ValidatesDistrolessContainer_Dataflow.yml

```diff
@@ -88,10 +88,17 @@ jobs:
           gcloud auth configure-docker us.gcr.io --quiet
           gcloud auth configure-docker gcr.io --quiet
           gcloud auth configure-docker us-central1-docker.pkg.dev --quiet
-      - name: run validatesDistrolessContainer script
+      - name: run validatesDistrolessContainer script (Java 17)
         uses: ./.github/actions/gradle-command-self-hosted-action
         with:
           gradle-command: :runners:google-cloud-dataflow-java:examplesJavaRunnerV2IntegrationTestDistroless
+          arguments: '-PtestJavaVersion=java17 -PdockerTag=$(date +%s)'
```

Comment:
`-PtestJavaVersion=java17` usually needs to be used along with `-Pjava17Home=...`. Otherwise the pipeline is still compiled and submitted in default Java (java 11) but run on java17 worker

Reply:
The distroless Dockerfile doesn't really care about Java. That Gradle property is needed to assign FROM using the referenced Beam Java SDK container image. Whatever anyone who designs or changes that original image, the distroless Dockerfile doesn't care.

Reply:
thanks, got it